onnx_xla/operator_registry.cc

```diff
@@ -34,4 +34,21 @@ TranslationMap& OperatorRegistry::map() {
   static TranslationMap map;
   return map;
 }
+
+std::vector<int64> OperatorRegistry::getMultidirectionalBroadcastArg(
+    const XlaBuilder& builder,
+    const XlaOp& firstOp,
+    const XlaOp& secondOp) {
+  auto firstNDim = ShapeUtil::Rank(builder.GetShape(firstOp).ValueOrDie());
+  auto secondNDim = ShapeUtil::Rank(builder.GetShape(secondOp).ValueOrDie());
+  std::vector<int64> broadcastDims;
+  if (firstNDim != secondNDim || firstNDim != 0 || secondNDim != 0) {
+    auto minDim = std::min(firstNDim, secondNDim);
+    auto maxDim = std::max(firstNDim, secondNDim);
+    for (auto j = 0; j < minDim; ++j) {
```

Comment:
nit: std::iota

Reply:
Have to resize first and then use iota so it does two passes?

Reply:
Yes, `resize` pads the space and `iota` fills in data until it hits the end of the iterator. You need both. But here you can just push. No need to go 2 pass.